src/sounde.F

```diff
@@ -75,15 +75,19 @@ subroutine sounde(dt,xh,arh1,arh2,uh,ruh,xf,uf,yh,vh,rvh,yf,vf,     &
 
 !---------------------------------------------------------------------
 
+    print *,'sounde: nsound: ',nsound
     IF( nrkmax.eq.3 )THEN
       if(nrk.eq.1)then
 !!!        nloop=1
 !!!        dts=dt/3.
```

Comment:
maybe just remove these two lines?

Reply:
This code was added to CM1 by another author.  No idea what it is still present